README.md

```diff
@@ -26,6 +26,16 @@ Regardless the user must have access to the repo being reviewed, and the token m
 config.handlers.reviewme.github_access_token = ENV["GITHUB_ACCESS_TOKEN"]
 ```
 
+The default message that is posted on GitHub pull request is `:eyes: @<reviewer>`.
+The message can be customized. **NOTE:** Be sure to add `%s` in order for the reviewer to be properly interpolated and tagged on GitHub.
```

Comment:
There another option for string formatting.

```ruby
"hi %{reviewer}" % {reviewer: "john"}
=> "hi john"
```

Is the explicitness worth a little extra syntax? I'm not sure one way or the other...